packages/protocol/contracts/L1/GuardianProver.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+
+pragma solidity ^0.8.20;
```

Comment:
@adaki2004 please move this file to /L1/provers/

Reply:
Ok, will do!